modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -0,0 +1,114 @@
+| `i3s.loadContent`  | Bool | `false` | Whether to load tile content (geometries, texture, etc.)                                                        |
+
+## Data formats
+
+This section specifies the loaded data formats.
+
+### Tileset Object
+
+The following fields are guaranteed. Additionally, the loaded tileset object will contain all the data fetched from the provided url.
+
+| Field            | Type     | Contents                                                                                                                                                                                                                                                                         |
+| ---------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `loader`         | `Object` | I3SLoader                                                                                                                                                                                                                                                                        |
+| `root`           | `Object` | The root tile header object                                                                                                                                                                                                                                                      |
+| `url`            | `String` | The url of this tileset                                                                                                                                                                                                                                                          |
+| `type`           | `String` | Value is `i3s`. Indicates the returned object is an `i3s` tileset.                                                                                                                                                                                                               |
+| `lodMetricType`  | `String` | Root's level of detail (LoD) metric type, which is used to decide if a tile is sufficient for current viewport. Only support `maxScreenThreshold` for now. Check I3S [lodSelection](https://github.com/Esri/i3s-spec/blob/master/docs/1.7/lodSelection.cmn.md) for more details. |
+| `lodMetricValue` | `Number` | Root's level of detail (LoD) metric value.                                                                                                                                                                                                                                       |
+
+### Tile Object
```

Comment:
Where do we stand on this, can we call this TileHeader or are you systematically renaming TileHeader to Tile?

Reply:
Choosing `Tile` from `TileHeader` is because `Tile`(or `TileHeader`) also has a `content` filed and fulfilled at some point, so `TileHeader` may not actually describe what it really contains.